### DISCUSSION TITLE
NEW EXPLOIT FOR INSERCURLY

### DISCUSSION THREAD

**Discussion by DLUNBOX** [3kh0/ext-remover#901] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Go to this web site and paste in this code
https://www.w3schools.com/html/tryit.asp?filename=tryhtml_default
[index (1).zip](https://github.com/3kh0/ext-remover/files/14113364/index.1.zip) Extract this

**Comment by Trentdamonke** [comment]
What code?

**Reply by DLUNBOX** [reply]
click the index file and extract it

**Reply by Trentdamonke** [reply]
I ran it and it says no adequate version or something. I have the newest version of securly that the extension id starts with a K

**Reply by DLUNBOX** [reply]
it worked for my so idk whats wrong

**Reply by DLUNBOX** [reply]
for me it does not say the K in the front

**Reply by Trentdamonke** [reply]
what is the id?

**Reply by DLUNBOX** [reply]
iheobagjkfklnlikgihanlhcddjoihkg

**Reply by Trentdamonke** [reply]
Yeah thats the old securly id before it was patched

**Reply by Trentdamonke** [reply]
is this not just insecurly?

**Reply by DLUNBOX** [reply]
i have no clue lol
